driver/src/main.rs

```diff
@@ -239,6 +239,11 @@ struct Options {
     /// target chain (e.g. WETH on mainnet, DAI on xDAI).
     #[structopt(long, env = "NATIVE_TOKEN_ID", default_value = "1")]
     native_token_id: u16,
+
+    /// Whether to ignore external price sources (e.g. 1Inch, Kraken etc)
+    /// when estimating token prices
+    #[structopt(long, env="IGNORE_EXTERNAL_PRICE_SOURCE", parse(try_from_str), default_value="false")]
+    ignore_external_price_source: bool,
```

Comment:
instead of ignore_... with default false we could avoid the negative and call it use_external_price_sources with default true. Not sure which I like it more, so just throwing it out there.

Reply:
👍 In the beginning I thought I could use the negative so that not specifying the flag means the defaults. But it turned out that this didn't work with ENV vars which we need in our kubernetes config. Changed it...